src/test/java/DukeTest.java

```diff
@@ -0,0 +1,44 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Deadline;
+import duke.DukeDeadlineException;
+import duke.DukeEventException;
+import duke.DukeTodoException;
+import duke.Event;
+import duke.Todo;
+
+public class DukeTest {
+    @Test
+    public void todoNormalTest() {
```

Comment:
Three part format `featureUnderTest_testScenario_expectedBehavior()` can be used for test method naming 😄